app/templates/INSTALL.mdown

```diff
@@ -112,39 +112,41 @@ When converting a project to pm2 you should create an `ecosystem.json` file with
 
 Now once you've almost finished deploying with pm2 you will need to stop it with the service scripts:
 
-`service stop {appname}`
+`sudo service stop {appname}`
 
 If you have a {appname}-watch script then stop that as well:
 
-`service stop {appname}-watch`
+`sudo service stop {appname}-watch`
 
 You should now be able to run `pm2 startOrRestart ecosystem.json`
 
 Now that the project is deployed and assuming it's running okay you will need to clean up the old deployment.
 
 Firstly, remove the chkconfig
 
-`chkconfig -del {appname}`
+`sudo chkconfig --del {appname}`
 
 And also for the watch script if appropriate
 
-`chkconfig -del {appname}-watch`
+`sudo chkconfig --del {appname}-watch`
 
 Then delete the scripts
 
-`rm /etc/init.d/{appname}`
+`sudo rm /etc/init.d/{appname}`
 
-`rm /etc/{appname}`
+`sudo rm -r /etc/{appname}`
 
 And for the watch script:
 
-`rm /etc/init.d/{appname}-watch`
+`sudo rm /etc/init.d/{appname}-watch`
+
+`sudo rm -r /etc/{appname}-watch`
```

Comment:
Hmm I had to use -r when removing the config for obi but it looks like just rm worked for alerts so it was probably just obi being a bit odd, I'll change it now